service/qos/raft_service_level_distributed_data_accessor.hh

```diff
@@ -41,6 +41,7 @@ public:
     virtual bool is_v2() const override;
     virtual bool can_use_effective_service_level_cache() const override;
     virtual ::shared_ptr<service_level_distributed_data_accessor> upgrade_to_v2(cql3::query_processor& qp, service::raft_group0_client& group0_client) const override;
+    static future<std::pair<utils::chunked_vector<mutation>, sstring>> set_service_level_mutations(cql3::query_processor& qp, sstring service_level_name, qos::service_level_options slo, api::timestamp_type timestamp);
```

Comment:
I am not sure if it is worth to complicate function return type so much only for debugging purposes, while sstring is actually hardcoded

Reply:
maybe just skip the whole cql statement in description, just logically write what those mutations are doing